ci/bugfix-release-pr-body.md

```diff
@@ -0,0 +1,63 @@
+AUTO: merge $RELEASE_BRANCH into $SOURCE_BRANCH
+
+This PR was automatically generated in response to a push to `master`,
+and is a chance to review any changes that will be included in the release
+branch before merging.  Under most circumstances, this PR will probably be
+a formality.  However, there are a few cases where we may need to do some
+extra work to make sure `$RELEASE_BRANCH` contains what it should after the
+merge:
+
+## There is a merge conflict in this PR
+
+1. Leave a comment on this PR about the merge conflict and close the PR.
+2. In your fork, make a new `pr-resolution` branch off of `$SOURCE_BRANCH`:
+   ```shell
+   $ git checkout $SOURCE_BRANCH
+   $ git pull upstream $SOURCE_BRANCH  # Include the latest changes on the upstream master
+   $ git checkout -b pr-resolution
+   $ git merge $RELEASE_BRANCH
```

Comment:
I'm not sure this ultimately makes a difference, but do we want to merge `release` into `master` or `master` into `release`... I guess I'm curious if the direction of the merge matters? In my head I say: "Master has been update, I want the new master commits in my release branch". Instead of "I want the release commits into the latest of master"... IDK, semantics probably.

Reply:
In either case, whether there's a merge conflict or there's something being PRed into the release branch that shouldn't be there, we want both `master` and `release/x` to be merged into `pr-resolution` so that we can take care of the issue.

So, master has been updated, and I want the new master commits to be in my release branch.  But just merging master into release is slightly wrong (either merge conflict or something isn't supposed to be included in the merge), so I need to handle the conflict.  Once the conflict has been resolved, then I want my fixed changes to be merged into the release branch.

```
     *  <-- This PR would be created once things are fixed in pr-resolution
    /|
   * | # Some changes to fix the conflict
   | |
   * <-- pr-resolution branch
  / \|
 /   * 
*    |
|    ^ release/x
^ master
``` 

Does that clarify things?  Or is there a better way to do this in git?

Reply:
Yes!